optimade/client/utils.py

```diff
@@ -116,6 +116,10 @@ def __init__(self):
             refresh_per_second=10,
         )
 
+    def print(self, *args, **kwargs):
```

Comment:
I do not think this print function is used by your code. 
In the OptimadeClient class, you overwrite the default behaviour of getattribute. 
See line 210 in client.py
So I think the print function of Rich gets used instead of this one. Which is why Codecov gives a warning here.